src/components/_charts/LineChart.tsx

```diff
@@ -10,9 +10,7 @@ const LineChart: VFC<LineSvgProps> = ({ data, ...rest }) => {
       enableGridX={false}
       enableGridY={false}
       tooltip={ToolTip}
-      motionConfig={{
-        duration: 0, // Disable animation upon changing views.
-      }}
+      motionConfig="default"
```

Comment:
I believe this was put in place due to a visual bug that would happen when switching from time resolutions. Will double-check this on the deploy preview

Reply:
edit: seems to be a non-issue!